tests/app/helpers/test_template_helpers.py

```diff
@@ -1,12 +1,109 @@
+                            "url": "https://census.gov.uk/help/how-to-answer-questions/online-questions-help/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "Contact us",
+                            "url": "https://census.gov.uk/contact-us/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "Languages",
+                            "url": "https://census.gov.uk/help/languages-and-accessibility/languages/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "BSL and audio videos",
+                            "url": "https://census.gov.uk/help/languages-and-accessibility/accessibility/accessible-videos-with-bsl/",
+                            "target": "_blank",
+                        },
+                    ]
+                }
+            ],
+            "legal": [
+                {
+                    "itemsList": [
+                        {
+                            "text": "Cookies",
+                            "url": "https://census.gov.uk/cookies/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "Accessibility statement",
+                            "url": "https://census.gov.uk/accessibility-statement/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "Privacy and data protection",
+                            "url": "https://census.gov.uk/privacy-and-data-protection/",
+                            "target": "_blank",
+                        },
+                        {
+                            "text": "Terms and conditions",
+                            "url": "https://census.gov.uk/terms-and-conditions/",
+                            "target": "_blank",
+                        },
+                    ]
+                }
+            ],
+        }
+
+        survey_config = CensusSurveyConfig()
+
+        result = ContextHelper(
+            language="en",
+            is_post_submission=False,
+            include_csrf_token=True,
+            survey_config=survey_config,
+        ).context["footer"]
+
+    assert result == expected
+
+
+def test_footer_warning_in_context_census_theme(app: Flask):
+    with app.app_context():
+        expected = "Make sure you <a href='/sign-out'>leave this page</a> or close your browser if using a shared device"
+
+        survey_config = CensusSurveyConfig()
+
+        result = ContextHelper(
+            language="en",
+            is_post_submission=True,
+            include_csrf_token=True,
+            survey_config=survey_config,
+        ).context["footer"]["footerWarning"]
+
+    assert result == expected
```

Comment:
The opposite test should also be done - the warning should not be present if `is_post_submission` is `False`.

Reply:
It's not tested explicitly, but it is tested in test_footer_context_census_theme indirectly, e.g. the comparison would fail if footerWarning was present. Obviously that isn't clear enough, so it would be better to have an additional test here for it